pusher-api/api-reference.md

```diff
@@ -0,0 +1,392 @@
+Pusher API reference
+=====================
+Pusher is a service that publishes information to the integrator's service. This information is data related to products, purchases, inventory etc. The purpose of this service is to ensure that integrators do not have to poll for data related to specific events. <br/>
+An integrator can subscribe to specific events in the Pusher service. When these events get triggered, Pusher service will publish information corresponding to the event to the integrator's service.
```

Comment:
An event occurs and triggers a callback(webhook) not gets triggered. 
I think most of webhook explanation can be copied from [PayPal docs](https://developer.paypal.com/docs/api-basics/notifications/webhooks/rest/), easy to read and understand.